js/src/tooltip.js

```diff
@@ -344,8 +346,6 @@ class Tooltip extends BaseComponent {
   setContent(content) {
     this._newContent = content
     if (this._isShown()) {
-      this.tip.remove()
```

Comment:
> This is done to reduce the bundle but I might be wrong. The 2 same lines are called in show() and I couldn't spot any use of this.tip between the actual removal and the removal in show(). So this way, this.tip is still removed but if it is there for the comprehension of the code it might be kept here

I think you are right on this